prime-router/metadata/fhir_mapping/hl7/resource/ServiceRequest.yml

```diff
@@ -104,6 +104,13 @@ identifier_4:
     code: "PGN"
     display: "Placer Group Number"
 
+
+identifier_5:
+  valueOf: datatype/Identifier_Gen
+  generateList: true
+  expressionType: resource
+  specs: ORC.21
+
```

Comment:
Do we need both the identifier and the extension under PractitionerRole for ORC.21?

Reply:
No, I can remove the extension, it did not seem to be working anyways. Good catch!